src/components/landing/listing.js

```diff
@@ -3,22 +3,42 @@ import Image from "next/image";
 import styles from "./listing.module.css";
 
 function Listing({ data }) {
-  const avatarWidth = 50
-  const avatarHeight = 50
+  const avatarWidth = 50;
+  const avatarHeight = 50;
   return (
     <>
       <article className={styles.listing}>
         <div className={styles.listing__image__container}>
           <Image
```

Comment:
Remove this `<Image />` because we are not getting this image from the managers.